Fastlane/deployment_lanes.rb

```diff
@@ -32,7 +32,7 @@
 
   if is_changed_since_last_tag == false
     tag_name = create_tag_name(xcodeproj: xcodeproj, target: target)
-    cancel_message = 'A new Release is cancelled as there are no changes since the last available tag.'
+    cancel_message = 'A new Beta build has been cancelled as there are no changes since the last available tag.'
```

Comment:
I think it'd be nice to (also) keep a reference to "release candidate" here; what do you think?

Reply:
Well the inspiration for this change was this slack conversation:
https://wetransfer.slack.com/archives/C3ZV6K8K0/p1655728927908339

The idea was to have a clear difference between what's "just" internal and what's going to be delivered to the customers. Therefore it was wished to not mention "releases" too much :)

Reply:
Gotcha — just don't want it to become confusing for us. 😄

Reply:
Same here. The build lane is already called "beta". I think it'll be much clearer for us if less things are called "release" ;)